library/alloc/src/vec.rs

```diff
@@ -50,8 +50,6 @@
 //! v[1] = v[1] + 5;
 //! ```
 //!
-//! [`Vec<T>`]: Vec
-//! [`new`]: Vec::new
 //! [`push`]: Vec::push
```

Comment:
I left this here since I don't know if removing this is better.

> //! You can [`push`] values onto the end of a vector (which will grow the vector
> //! as needed):

Maybe?

> You can push values onto the end of a vector with [`Vec::push`] (which will grow
> the vector as needed):

Reply:
I think it's better to be concise then to repeat ourselves in the docs to avoid repeating code. The second one _works_ but I like the first better.

Reply:
Understandable, I do think the first one is better.